api/apps/api/src/modules/projects/projects-crud.service.ts

```diff
@@ -250,7 +264,15 @@ export class ProjectsCrudService extends AppBaseService<
     fetchSpecification: FetchSpecification,
     info?: ProjectsInfoDTO,
   ): Promise<SelectQueryBuilder<Project>> {
+    assertDefined(info?.authenticatedUser);
```

Comment:
(I am aware that it will need to be re-implemented a bit for a `public` projects feature)